### DISCUSSION TITLE
Welcome to RhodoniteTS Discussions!

### DISCUSSION THREAD

**Discussion by emadurandal** [actnwit/RhodoniteTS#928] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  Note: If you have a specific and more definite topic, please create a Github Issue instead of here.

**Comment by cx20** [comment]
I have been experimenting with WebGL libraries as part of my hobby.
The `Rhodonite` sample I used in my tests can be found in the repository below.
https://github.com/cx20/gltf-test
https://github.com/cx20/webgl-test
https://github.com/cx20/webgl-physics-examples

**Reply by emadurandal** [reply]
@cx20 Thank you so much! I'll do my best to improve Rhodonite hardly!
Feel free to ask me anything for evaluating Rhodonite.
